### PR TITLE
feat: Add flash messages

### PR DESCRIPTION
**Pull request by hrushikesh-git**
@pnijhara Added flash messages to the post adding and deleting and login and logout. Please review the code and merge.

### CODE REVIEW

**Comment by pnijhara** [comment]
@hrushikesh-git most of the changes that you have made are similar to the one that you have done in #46. Like importing flash in `app.js`. Can you please re-look at them?

File: app.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const expressSession = require("express-session")
 const LocalStrategy = require("passport-local")
 const methodOverride = require("method-override")
 const foodData = require("./models/foodup")
+const flash = require("connect-flash")
```
Comment (pnijhara) [inline app.js:11]: This you have already added in #46

File: app.js [hunk 2]
```diff
@@ -45,6 +46,8 @@ app.use(expressSession({
 }))
 
 
+app.use(flash());
```
Comment (pnijhara) [inline app.js:49]: This also!

**Comment by pnijhara** [comment]
@hrushikesh-git also make sure your current branch is upto-date with the master branch of this project

**Comment by hrushikesh-git** [comment]
Okay, I guess that happened because my current branch was not updated as per the master branch. Do I have to fork the new master branch again ?? how should I update my current branch ??

**Comment by pnijhara** [comment]
You do not need to fork the master every time. You just need to pull the changes. Try pulling the master branch in your current branch and then push it. This will (most probably) update your PR.

**Comment by pnijhara** [comment]
@hrushikesh-git this new commit of yours should have gone in a new PR. This PR was more towards flash messages. I request you to please revert this commit and look at the changes that I have suggested above.

**Comment by hrushikesh-git** [comment]
> @hrushikesh-git this new commit of yours should have gone in a new PR. This PR was more towards flash messages. I request you to please revert this commit and look at the changes that I have suggested above.

Brother, I am not getting this I mean I am confused what should I do ??

**Comment by pnijhara** [comment]
It is simple you need to just move back 1 commit and push it again. So that I can merge this PR.
The changes you have made in the latest commit, make them again in a new branch and send a seperate PR for the same(improved UI).

**Pull request closed without merging**